Makefile

```diff
@@ -481,7 +483,27 @@ docs: .watch-docs dist/docs
 
 all: init test-js-remote test-py-all test-scala test-r sdist install-all system-test
 
+export-release-version:
+	docker $(DOCKER_OPTS) run -it --rm \
+		-v `pwd`/dist:/src \
+		-v `pwd`/etc/read_release_version.py:/src/read_release_version.py \
+		$(REPO) bash -c 'python /src/read_release_version.py' > `pwd`/dist/RELEASE_VERSION
+
+publish-scala-jar: export-release-version
```

Comment:
I think this is missing a dependency... for it to be a standalone, shouldn't it depend on the recipe that builds scala